customer-onboarding-showcase/resources/.camunda/element-templates/evaluate-risk-01.json

```diff
@@ -0,0 +1,92 @@
+{
```

Comment:
I was about to add an issue for the formatting of the element template. Horrible! And that is the default jackson beautifier. That's on me ofc

Reply:
Should we open an issue for that or does it just look weird and annoying to me?